shell-completion/zsh/_systemd-inhibit

```diff
@@ -33,6 +33,6 @@ _arguments \
     '--what=[Operations to inhibit]:options:_systemd-inhibit_what' \
     '--who=[A descriptive string who is inhibiting]:who is inhibiting:' \
     '--why=[A descriptive string why is being inhibited]:reason for the lock:' \
-    '--mode=[One of block or delay]:lock mode:( block delay )' \
+    '--mode=[One of block, block-weak, delay or delay-weak]:lock mode:( block block-weak delay delay-weak )' \
```

Comment:
Oxford comma please.

The same change needs to be done in `--help` output for systemd-inhibit.